#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From a1afa2a58ea2b25887b5f9124cf5c886e799e547 Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@intevation.de>
Date: Thu, 28 Jan 2016 18:22:07 +0100
Subject: [PATCH] Make DBus optional

DBus is only used for the icon changed notifications emited by
KIconLoader.
---
 CMakeLists.txt                 | 10 +++++++++-
 src/CMakeLists.txt             |  7 ++++++-
 src/config-kiconthemes.h.cmake |  2 ++
 src/kiconloader.cpp            | 15 +++++++++++++--
 4 files changed, 30 insertions(+), 4 deletions(-)
 create mode 100644 src/config-kiconthemes.h.cmake

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5664ea1..22642b9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,7 +36,7 @@
 set(REQUIRED_QT_VERSION 5.10.0)
 find_package(Qt5Widgets ${REQUIRED_QT_VERSION} REQUIRED NO_MODULE)
 find_package(Qt5Svg ${REQUIRED_QT_VERSION} REQUIRED NO_MODULE)
-if (NOT ANDROID)
+if (NOT ANDROID AND NOT WIN32)
   find_package(Qt5DBus ${REQUIRED_QT_VERSION} REQUIRED NO_MODULE)
 endif()

--
2.1.4
